modules/local/tree_summary.nf

```diff
@@ -0,0 +1,50 @@
+process TREE_SUMMARY {
+    tag "$meta.id"
+    label 'process_single'
+
+    container = 'ecoflowucl/genomeqc_tree:v1.3'
+    publishDir "$params.outdir/tree_plots" , mode: "${params.publish_dir_mode}", pattern:"Phyloplot_*.pdf"
+
+    input:
+    tuple val(meta), path(tree)
+    path  multiqc_files
+
+    output:
+    path( "P*.pdf"          ),                emit: figure
+    path( "versions.yml"    ),                emit: versions
+
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+
+    #Remove unwanted extensions in the tree file
+    sed \'s/\\.prot\\.fa\\.largestIsoform//g\' ${tree}/Species_Tree/SpeciesTree_rooted_node_labels.txt > tree.nw
+    
+    # Combine the BUSCO outputs
+    head -qn 1 *.txt | head -n 1 > Busco_combined
+    tail -q -n 1 *.txt          >> Busco_combined
+    cut -f 1,3,4,5,6,7 Busco_combined >> Busco_combined_cut
+    sed -i \'s/\\.fasta//g\' Busco_combined_cut
+
+    python3 ${projectDir}/bin/busco_2_table.py Busco_combined_cut Busco_to_plot.tsv
```

Comment:
If you make the scripts executable, then you don't need to call the files in this way. Just make sure the interpreter is in the shebang line.

Reply:
https://github.com/nf-core/genomeqc/pull/66

Reply:
This PR removes this issue